ietf/meeting/models.py

```diff
@@ -684,6 +684,11 @@ def owner_email(self):
     def is_official(self):
         return (self.meeting.schedule == self)
 
+    @property
+    def is_official_record(self):
```

Comment:
I will keep that in mind.  You are correct in that I was just cargo-culting the `is_official()` method above.

Do you know why one would wish to avoid the `@property` decorator?  Does it come with some sort of cost that isn't obvious to me?

Reply:
Mainly it masks the fact that a possibly expensive method is being called.